docs/spec/operations.md

```diff
@@ -42,14 +42,18 @@ Use the following process to generate a Sidetree-based DID:
       "recoveryCommitment": COMMITMENT_HASH
     }
     ```
-    - The object ****MAY**** contain a `type` property, and if present, its value ****MUST**** be a type string, of a length and composition defined by the implementation, that signifies the type of entity a DID represents. The value ****MUST NOT**** be anything that represents humans, groups of humans, or any human-identifying classification - an implementation ****SHALL NOT**** override this directive, and any attempt to do so shall render the implementation non-conformant to this specification.
+    - The object ****MAY**** contain a `type` property, and if present, its value ****MUST**** be a type string, of a length and composition defined by the implementation, that signifies the type of entity a DID represents.
     - The object ****MUST**** contain a `deltaHash` property, and its value ****MUST**** be a hash of the canonicalized [_Create Operation Delta Object_](#create-delta-object) (detailed above), generated via the [`HASH_PROTOCOL`](#hash-protocol).
     - The object ****MUST**** contain a `recoveryCommitment` property, and its value ****MUST**** be the [recovery commitment](#recovery-commitment) as generated in step 2.
 
 ::: note
 Implementations ****MAY**** choose to define additional properties for inclusion in the [_Create Operation Suffix Data Object_](#create-suffix-data-object), but the presence of any properties beyond the standard properties or implementation-defined properties ****ARE NOT**** permitted.
 :::
 
+::: warning
+The type value must be carefully considered to avoid anything that represents humans, groups of humans, or any human-identifying classification.
```

Comment:
Suggested text: "The string values used in the `type` field must be carefully considered, and this specification strongly cautions implementers to avoid allowing any values that represent humans, groups of humans, or any human-identifying classifications."

Reply:
@csuwildcat Done: https://github.com/decentralized-identity/sidetree/compare/d49d044c0d1f7ea926c514828c829078be27dfc6..9e106b15a317a9c731f1ad37be8cfb3c5a6c91dc